MoneyAPP/moneyapp/lib/blocs/transactions/transactions_bloc.dart

```diff
@@ -24,15 +24,15 @@ class TransactionsBloc extends Bloc<TransactionsEvent, TransactionsState> {
       date: DateTime.now(),
     );
 
-    //push back
+    
     final updatedTransactions = List<Transaction>.from(state.transactions)
       ..add(newTransaction);
 
     final updatedBalance = state.balance +
         (event.isTopUp
             ? event.amount
             : -event
-                .amount); //this will be implemented further on, basicly a if statment
+                .amount); 
```

Comment:
Please autoformat the code. Enable format on save. 

<img width="1187" alt="Screenshot 2024-10-07 at 16 45 41" src="https://github.com/user-attachments/assets/047b292a-9952-4c16-912c-19871c3995f1">

Reply:
Fixed.